install/kubernetes/helm/istio/values.yaml

```diff
@@ -358,7 +358,16 @@ global:
   #   services or ServiceEntries for the destination port
   outboundTrafficPolicy:
     mode: REGISTRY_ONLY
-  
+
+  # configRootNamespace: istio-config
```

Comment:
Shouldn't we be able to have it on by default, and if no Sidecar is present in that NS keep the current behavior ? One less setting step for the user.

Reply:
ack